src/MassTransit/Monitoring/LogContextInstrumentationExtensions.cs

```diff
@@ -48,270 +62,293 @@ public static class Instrumentation
                 return null;
 
             var tagList = new TagList
             {
-                { _options.ServiceNameLabel, _serviceName },
-                { _options.ActivityNameLabel, context.ActivityName },
-                { _options.ArgumentTypeLabel, GetArgumentTypeLabel<TArguments>() }
+                { _options.ServiceNameLabel, _options.ServiceName },
+                { _options.MessageTypeLabel, GetMessageTypeLabel<T>() }
             };
 
-            return TrackInProgress(_executeInProgress, tagList);
+            _outboxSendTotal.Add(1, tagList);
+
+            return new StartedInstrument(exception =>
+            {
+                tagList.Add(_options.ExceptionTypeLabel, exception.GetType().Name);
+                _outboxSendFaultTotal.Add(1, tagList);
+            });
         }
 
-        public static IDisposable TrackCompensateActivityInProgress<TActivity, TLog>(CompensateActivityContext<TActivity, TLog> context)
-            where TActivity : class, ICompensateActivity<TLog>
-            where TLog : class
+        public static StartedInstrument? StartOutboxDeliveryInstrument(this ILogContext logContext, OutboxMessageContext context)
         {
-            if (!_compensateTotal.Enabled)
+            if (!_isConfigured || !_outboxDeliveryTotal.Enabled)
                 return null;
 
             var tagList = new TagList
             {
-                { _options.ServiceNameLabel, _serviceName },
-                { _options.ActivityNameLabel, context.ActivityName },
-                { _options.LogTypeLabel, GetLogTypeLabel<TLog>() }
+                { _options.ServiceNameLabel, _options.ServiceName },
+                { _options.EndpointLabel, GetEndpointLabel(context.DestinationAddress) }
             };
 
-            return TrackInProgress(_compensateInProgress, tagList);
-        }
+            _outboxDeliveryTotal.Add(1, tagList);
 
-        public static IDisposable TrackHandlerInProgress<TMessage>()
-            where TMessage : class
-        {
-            var tagList = new TagList
+            return new StartedInstrument(exception =>
             {
-                { _options.ServiceNameLabel, _serviceName },
-                { _options.MessageTypeLabel, GetMessageTypeLabel<TMessage>() },
-            };
-
-            return TrackInProgress(_handlerInProgress, tagList);
+                tagList.Add(_options.ExceptionTypeLabel, exception.GetType().Name);
+                _outboxDeliveryFaultTotal.Add(1, tagList);
+            });
         }
 
-        static IDisposable TrackInProgress(Counter<long> counter, TagList tagList)
+        public static void TryConfigure(IServiceProvider provider)
         {
-            counter.Add(1, tagList);
+            if (_isConfigured)
+                return;
 
-            return new InProgressTracker(counter, tagList);
+            TryConfigure(provider.GetRequiredService<IOptions<InstrumentationOptions>>().Value);
```

Comment:
This is _always_ going to enable instrumentation, which is _not_ a good thing. It should only be configured if it's actually configured.